include/sass/values.h

```diff
@@ -35,6 +35,13 @@ enum Sass_Separator {
   SASS_HASH
 };
 
+  // Tags for denoting Sass list delimiters
+enum Sass_List_Delimiter {
```

Comment:
I'm not too excited about the wording "delimiter". I only use delimiter for stuff that goes between things and not around. Maybe that's just me? But I would have preferred something as `Sass_List_Type`. IMO it is confusing to have a `delimiter` and a `separator`.